GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -287,8 +287,17 @@ internal FormBrowse(GitUICommands commands, BrowseArguments args, ISettingsSourc
                 PluginRegistry.Initialize();
                 await ThreadHelper.JoinableTaskFactory.SwitchToMainThreadAsync();
                 RegisterPlugins();
-                revisionDiff.RegisterGitHostingPluginInBlameControl();
-                fileTree.RegisterGitHostingPluginInBlameControl();
+                try
+                {
+                    revisionDiff.RegisterGitHostingPluginInBlameControl();
+                    fileTree.RegisterGitHostingPluginInBlameControl();
+                }
+                catch (Exception)
```

Comment:
Limit to ExternalException to only ignore GE reraised?
(or similar strategy)
The may be other exceptions due to the implementation that should be handled.